tasks/google/cloud/tasks_v2beta2/types.py

```diff
@@ -26,41 +32,23 @@
 from google.protobuf import timestamp_pb2
 from google.rpc import status_pb2
 
-from google.api_core.protobuf_helpers import get_messages
-from google.cloud.tasks_v2beta2.proto import cloudtasks_pb2
-from google.cloud.tasks_v2beta2.proto import queue_pb2
-from google.cloud.tasks_v2beta2.proto import target_pb2
-from google.cloud.tasks_v2beta2.proto import task_pb2
-
-
-_shared_modules = [
-    http_pb2,
-    iam_policy_pb2,
-    policy_pb2,
-    any_pb2,
-    descriptor_pb2,
-    duration_pb2,
-    empty_pb2,
-    field_mask_pb2,
-    timestamp_pb2,
-    status_pb2,
-]
-
-_local_modules = [
-    cloudtasks_pb2,
-    queue_pb2,
-    target_pb2,
-    task_pb2,
-]
-
 names = []
-
-for module in _shared_modules:
-    for name, message in get_messages(module).items():
-        setattr(sys.modules[__name__], name, message)
-        names.append(name)
-
-for module in _local_modules:
+for module in (
+        http_pb2,
+        cloudtasks_pb2,
+        queue_pb2,
+        target_pb2,
+        task_pb2,
+        iam_policy_pb2,
+        policy_pb2,
+        any_pb2,
+        descriptor_pb2,
+        duration_pb2,
+        empty_pb2,
+        field_mask_pb2,
+        timestamp_pb2,
+        status_pb2,
+):
```

Comment:
The generator needs to be fixed -- it should not be undoing the `_shared_modules` / `_local_modules` split.

Reply:
@theacodes can you confirm that this is making an incorrect change? I can add a transform to handle it if so.

Reply:
PS - Vision gen has the same issue if this is a problem

Reply:
See #5364.

Reply:
addressed.